### PR TITLE
[ATIP] Reconstruct aitp android uiautomator model core

### PR DESCRIPTION
**Pull request by jacky-young**
-Reconstruct aitp android uiautomator model core

BUG=https://crosswalk-project.org/jira/browse/XWALK-4781

### CODE REVIEW

**Comment by jacky-young** [comment]
@Honry please review it.

File: tools/atip/README.md [hunk 1]
```diff
@@ -414,49 +386,28 @@ def i_visit_url(context, url):
...
-@step(u'I process any object "{class_name}" "{value_name}"')
-@step(u'I process "{class_name}" on the "{relative}" side of text "{text_name}"')
-@step(u'I process "{class_name}" on the "{relative}" side of view "{view_desc}"')
-@step(u'I process "{class_target}" on the "{relative}" side of any "{class_name}" "{value_name}"')
-@step(u'I reload process result to temporary value "{key}"')
+@step(u'I should see view "{params_kw}"')
+@step(u'I should see relative view "{params_kw1}" on the "{relative}" side of view "{params_kw2}"')
+@step(u'I should not see view "{params_kw}"')
+@step(u'I should not see relative view "{params_kw1}" on the "{relative}" side of view "{params_kw2}"')
+@step(u'I click view "{button_name}"')
```
Comment (Honry) [inline tools/atip/README.md]: Why the param name is button_name while you click is view?

File: tools/atip/README.md [hunk 2]
```diff
@@ -414,49 +386,28 @@ def i_visit_url(context, url):
...
+@step(u'I click object by key "{key}"')
```
Comment (Honry) [inline tools/atip/README.md]: Please keep step name consistent with above one.

File: tools/atip/README.md [hunk 3]
```diff
@@ -414,49 +386,28 @@ def i_visit_url(context, url):
...
+@step(u'I edit view "{params_kw}" to input "{text}"')
+@step(u'I edit index {n:d} EditText to input "{text}"')
+@step(u'I compare view "{params_kw}" info "{what}" with "{except_result}"')
```
Comment (Honry) [inline tools/atip/README.md]: 's/what/info_name'

File: tools/atip/README.md [hunk 4]
```diff
@@ -414,49 +386,28 @@ def i_visit_url(context, url):
...
+@step(u'I compare object by key "{key1}" equal "{key2}" on info "{what}"')
+@step(u'I compare object by key "{key1}" unequal "{key2}" on info "{what}"')
+@step(u'I save view "{params_kw}" to object by key "{key}"')
+@step(u'I save relative view "{params_kw1}" on the "{relative}" side of view "{params_kw2}" to object by key "{key}"')
```
Comment (Honry) [inline tools/atip/README.md]: 's/relative/position'

File: tools/atip/README.md [hunk 5]
```diff
@@ -414,49 +386,28 @@ def i_visit_url(context, url):
...
+@step(u'I save finding view "{params_kw1}" on the "{relative}" side of view "{params_kw2}" process')
```
Comment (Honry) [inline tools/atip/README.md]: I save process of finding view ....

File: tools/atip/README.md [hunk 6]
```diff
@@ -414,49 +386,28 @@ def i_visit_url(context, url):
...
+@step(u'I reload above process and save result to object by key "{key}"')
+@step(u'I wait object by key "{key}" exist for "{time_out}"')
```
Comment (Honry) [inline tools/atip/README.md]: for "{time_out}" / in"{time_out}" seconds

File: tools/atip/atip/android/steps.py [hunk 7]
```diff
@@ -62,196 +62,87 @@ def register_watcher_when2(context, watcher_name, when_text1, when_text2, click_
...
-def select_relative_text_object(context, class_name, relative, text_name):
-    ob = context.android.selectTvObjectBy(text_name)
-    assert ob.exists
-    assert context.android.selectRelativeObjectBy(ob, relative, class_name).exists
-
-
-@step(u'I should see class "{class_name}" on the "{relative}" side of view "{view_desc}"')
-def select_relative_view_object(context, class_name, relative, view_desc):
-    ob = context.android.selectViewObjectBy(view_desc)
+@step(u'I should see view "{params_kw}"')
```
Comment (Honry) [inline tools/atip/atip/android/steps.py]: Please add note to describe the step.

**Comment by jacky-young** [comment]
tested locally

**Comment by Honry** [comment]
LGTM

**Comment by Honry** [comment]
@wanghongjuan, please notice this update, and you have to update your steps name in bdd tests.

**Pull request merged**